core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/MetricNameService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics.core.dropwizard;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricFilter;
+import com.google.common.base.Preconditions;
+
+/**
+ * Provides some basic functions for DropWizard metric names. In order to avoid naming collisions, each metric name
+ * is prefixed with the hostname, which defaults to the hostname of the local host. The format is,
+ * <code>hostname:metric_name</code>. All metric names should be created using {@link #createMetricName(String)} in
+ * order to ensure a consistent format.
+ *
+ * @author jsanda
+ */
+public class MetricNameService implements MetricFilter {
+
+    public static final String TENANT_ID = "org.hawkular.metrics";
```

Comment:
The original design document for admin features does not name an admin tenant. Two obvious choices are `admin` and `metrics-admin`. I prefer first since it is shorter and simpler, but the second one is more useful if Alerting will want their own admin tenant too.